controllers/get-api-topics.controller.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection")
+const {selectTopics} = require("../models/get-api-topics.model")
+
+const getTopics = (req, res) => {
+    
+ return selectTopics().then((data) =>{
+    const response = {}
+    response.msg = "200 - request successful"
```

Comment:
There is no need to send this message - sending the status code is adequate enough for all endpoints as this is interpreted without a message. This status code along with the correct data on an appropriate key on the response body (eg "topics") is all this endpoint needs to return.